style.css

```diff
@@ -581,3 +587,468 @@ a {
   margin-left: 100px;
   margin-bottom: -18px;
 }
+
+@media all and (min-width: 768px) {
+  nav {
+    margin-right: 110px;
+    font-weight: 272;
+  }
+
+  nav i {
```

Comment:
Your page is not very responsive. The page does not `shrink` and `grow` accordingly on different view widths. Kindly try to use `vw` or `%` for your widths to make your page responsive (i.e `width: 100vw;` or `width: 100%;`). The `px` makes widths rigid and unresponsive.

This is what happens from different viewpoints:
on about 1748px viewpoint:
![image](https://user-images.githubusercontent.com/38088818/165939238-370bda1a-d8ff-4391-861d-13ff3765c612.png)

on about 784px to 1228px viewpoint:
![image](https://user-images.githubusercontent.com/38088818/165939483-211ae0c6-88e7-4d45-b0c4-c04e406ed104.png)

![image](https://user-images.githubusercontent.com/38088818/165939907-c9435623-01d6-482f-91a0-fa11eb94426e.png)


Kindly inspect your page and use the different viewpoints to debug the problem: